judge/models/profile.py

```diff
@@ -125,16 +124,10 @@ def calculate_points(self, table=(lambda x: [pow(x, i) for i in xrange(100)])(ge
 
     @cached_property
     def display_name(self):
-        if self.name:
-            return self.name
         return self.user.username
 
     @cached_property
     def long_display_name(self):
```

Comment:
Hopefully we can get rid of this too.

Reply:
> WIP

Reply:
Of course, but this should be part of the work that is currently in progress.